src/oaklib/interfaces/text_annotator_interface.py

```diff
@@ -95,14 +95,19 @@ def annotate_text(
                         yield ann
 
     def annotate_file(
-        self, text_file: TextIOWrapper, configuration: TextAnnotationConfiguration = None
+        self,
+        text_file: TextIOWrapper,
+        terms_to_remove: List[str],
```

Comment:
let's not extend the signature of the method, this is not sustainable, if we keep doing this it will be a big ad-hoc set of key-values. Instead, extend the configuration data model

Reply:
When you say configuration data model, what does that mean?

Reply:
Makes sense, sorry for the oversight.